docs/docs/reference-react.md

```diff
@@ -1,24 +1,67 @@
 ---
-id: reference-react
+id: react-api
 title: React Top-Level API
 layout: docs
 category: Reference
-permalink: docs/reference-react.html
+permalink: docs/react-api.html
 redirect_from: "/docs/reference.html"
 ---
 
 `React` is the entry point to the React library.
 
-## Reference
+## Overview
+
+### Components
+
+React components let you split the UI into independent, reusable pieces, and think about each piece in isolation. React components can be defined by subclassing `React.Component`.
 
  - [`React.Component`](#react.component)
- - [`createClass()`](#createclass)
- - [`createElement()`](#createlement)
- - [`cloneElement()`](#cloneelement)
+ - [`createClass()`](#react.createclass)
+
+#### Typechecking with PropTypes
+
```

Comment:
I'd like to see typechecking at the bottom, not before createElement. While you don't use createElement in normal code it's best to understand what it is. PropTypes are the only API completely unrelated to everything else, and I think they should come last.